From: Keith Packard <keithp@keithp.com>
Date: Thu, 1 Nov 2018 13:21:05 -0700
Subject: Fix dependencies for debian build.

Removed prawn/templates, which eliminates the ability to use .pdf
files as images.

Signed-off-by: Keith Packard <keithp@keithp.com>
---
 asciidoctor-pdf.gemspec                    | 6 +++---
 lib/asciidoctor-pdf/prawn-templates_ext.rb | 2 +-
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/asciidoctor-pdf.gemspec b/asciidoctor-pdf.gemspec
index 6796786..3d430ce 100644
--- a/asciidoctor-pdf.gemspec
+++ b/asciidoctor-pdf.gemspec
@@ -38,14 +38,14 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency 'asciidoctor', '>= 1.5.3', '< 3.0.0'
   s.add_runtime_dependency 'prawn', '~> 2.2.0'
   s.add_runtime_dependency 'prawn-table', '~> 0.2.0'
-  s.add_runtime_dependency 'prawn-templates', '~> 0.1.0'
+  # s.add_runtime_dependency 'prawn-templates', '~> 0.1.0'
   s.add_runtime_dependency 'prawn-svg', '~> 0.29.0'
   s.add_runtime_dependency 'prawn-icon', '~> 2.3.0'
   s.add_runtime_dependency 'safe_yaml', '~> 1.0.0'
   s.add_runtime_dependency 'thread_safe', '~> 0.3.0'
-  s.add_runtime_dependency 'concurrent-ruby', '~> 1.1.0'
+  s.add_runtime_dependency 'concurrent-ruby', '~> 1.0.5'
   # For our usage, treetop 1.6 is slower than treetop 1.5
-  s.add_runtime_dependency 'treetop', '~> 1.5.0'
+  s.add_runtime_dependency 'treetop', '>= 1.5.0'
 
   s.add_development_dependency 'rake', '~> 12.3.0'
   # Asciidoctor PDF supports Rouge >= 2; Rouge 3.4.1 emits a superfluous warning in verbose mode
diff --git a/lib/asciidoctor-pdf/prawn-templates_ext.rb b/lib/asciidoctor-pdf/prawn-templates_ext.rb
index fca8178..92fe260 100644
--- a/lib/asciidoctor-pdf/prawn-templates_ext.rb
+++ b/lib/asciidoctor-pdf/prawn-templates_ext.rb
@@ -2,4 +2,4 @@ class Prawn::Document
   # NOTE allows prawn-templates 0.0.4 to be used with prawn >= 2.2.0
   const_set :VALID_OPTIONS, (send :remove_const, :VALID_OPTIONS).dup if VALID_OPTIONS.frozen?
 end
-require 'prawn/templates'
+# require 'prawn/templates'
